src/lib.rs

```diff
@@ -264,6 +273,21 @@ fn try_convert_duration_to_go_type(from: Duration) -> Result<i64, StartError> {
     i64::try_from(from.as_nanos()).map_err(|_| StartError::DurationIsTooLong(from))
 }
 
+// Why we are disabling clippy::missing_panics_docs: This function never panics. The only `unwrap()`
+// call is on the line handling the `None` case and we know it's not going to panic because "" does
+// not contain any null bytes.
```

Comment:
```suggestion
// The only `unwrap()` call is handling the `None` case and can't panic (see below)
```

Maybe this is sufficient?

Reply:
I decided to simplify this code and remove `unwrap`. The downside: an extra allocation when there is no AccessToken set. I think code clarity is more important than micro-optimization here.

Reply:
See [5d32868](https://github.com/filecoin-station/rusty-lassie/pull/32/commits/5d32868a7a32ac64db8c719873e7990ebfc0f869)